### DISCUSSION TITLE
Recommended Specs for hosting

### DISCUSSION THREAD

**Discussion by ede1998** [searxng/searxng#3884] [Q&A] [answered]
Hey everyone!
I'm considering to host my own instance and was just wondering what the requirements for SearxNG are. For instance, RAM, CPU, storage. Minimum and/or recommended.

The admin documentation does not seem to mention it.

**Comment by vojkovic** [comment, marked as answer]
SearXNG is extremely lightweight. While I don't have any official 'recommended' number, I can say I personally host SearXNG on a few 1vcpu, 512mb vms and have had 0 problems with speed. The docker container takes up about 300mb of space.

**Reply by ede1998** [reply]
Thank you. I was unsure about space thinking it might do some caching because the docker apparently uses a Redis instance.

**Reply by vojkovic** [reply]
Yup - the redis db is for some things like the limiter and also is used for some engines like duckduckgo. Search results never get cached. The size of the redisdb is only about like 10mb... :)

**Reply by return42** [reply]
I've just merged the [Favicons](https://docs.searxng.org/admin/searx.favicons.html) ... if you activate them (but first read the doc!) you should plan for a larger cache:

- https://github.com/searxng/searxng/pull/3727#issuecomment-2370980820
- https://github.com/searxng/searxng/pull/3727#issuecomment-2392068214
- https://github.com/searxng/searxng/pull/3727#issuecomment-2392858799

**Reply by catfluoride** [reply]
> I've just merged the [Favicons](https://docs.searxng.org/admin/searx.favicons.html) ... if you activate them (but first read the doc!) you should plan for a larger cache:
> 
> * [[feat] add favicons to result urls #3727 (comment)](https://github.com/searxng/searxng/pull/3727#issuecomment-2370980820)
> * [[feat] add favicons to result urls #3727 (comment)](https://github.com/searxng/searxng/pull/3727#issuecomment-2392068214)
> * [[feat] add favicons to result urls #3727 (comment)](https://github.com/searxng/searxng/pull/3727#issuecomment-2392858799)

I tried to activate favicons on my docker instance by adding the relevant entry in the "server" section (as per docs). But  I guess the docker image still don't support them?

**Reply by return42** [reply]
> But I guess the docker image still don't support them?

We still waiting for a fix of https://github.com/searxng/searxng/issues/3887

**Reply by catfluoride** [reply]
> > But I guess the docker image still don't support them?
> 
> We still waiting for a fix of #3887

Thank you for the update :-)
